src/smart-components/App/stories/index.stories.js

```diff
@@ -376,6 +376,24 @@ const CustomApp = () => {
 
 export const customer1 = () => fitPageSize(<CustomApp />);
 
+export const customer2 = () => (
```

Comment:
Can you make it obvious(rename customer2 to something else) so that its easier to understand this is a page with scroll?